pkg/download/blob_test.go

```diff
@@ -64,7 +64,7 @@ func Test_blobDownload_getURL(t *testing.T) {
 }
 
 func Test_blobDownload_fails_badCreds(t *testing.T) {
-	d := NewBlobDownload("accountname", "Zm9vCg==", blobutil.AzureBlobRef{
+	d := NewBlobDownload("example", "Zm9vCg==", blobutil.AzureBlobRef{
```

Comment:
Yes it may hit this again but it's a real-world test actually making a call to storage, so I tend to keep it for now.